data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Bactrian Unlock [0]"
+	minor
+	name "Locate Missing Scientists"
+	description "A Deep Star Queen transporting top scientists has gone missing. It was last seen near the Betelgeuse system in the Far North. Find its whereabouts and return it to <destination> if possible."
+	source
+		attributes deep
+	destination Valhalla
+	waypoint Betelgeuse
+	to offer
+		has "Smuggler's Den, Part 2: done"
+		has "Expedition to Hope 2: done"
+		has "WR Star 1: done"
+		has "Rescure Miners 2: done"
+		has "Humanitarian 2: done"
+		has "Terraforming 5: done"
+	on offer
+		conversation
+			`While you are watching one of the telescreens in a restaurant located in the spaceport, you are approached by a man and a woman, both wearing Deep Security uniforms.`
+			`	"Hello. Are you <first> <last>?" the man asks`
+			choice
+				`	"Yes I am. Is there something you need?"`
+					goto named
+				`	"Sorry, but I'm not the person you're looking for."`
+				
+			`	The man looks a little embarrassed after you say that. "Alright. Sorry for bothering you." The two make their way out of the restaurant and you return to watching the telescreen.`
+				decline
+	
+			label named
+			`	"Good, because we were wondering if you would be able to assist us. We have a record of you having assisted in a number of humanitarian ventures before."`
```

Comment:
Idk. I think it would feel more natural if you removed the "Good, because". It's not like it contributes anything to the conversation; when you say "yes" you already establish that you are `<first> <last>`. It doesn't make much difference, if Derpy's really stubborn he could keep it, but it doesn't really serve any purpose there.